src/test/java/org/opensearch/neuralsearch/query/aggregation/BaseAggregationsWithHybridQueryIT.java

```diff
@@ -0,0 +1,253 @@
+    protected static final String NESTED_FIELD_1_VALUE_4 = "sun";
+    protected static final String NESTED_FIELD_2_VALUE_4 = "wukong";
+    protected static final String NESTED_FIELD_1_VALUE_5 = "vasilisa";
+    protected static final String NESTED_FIELD_2_VALUE_5 = "the wise";
+    protected static final String INTEGER_FIELD_1 = "doc_index";
+    protected static final int INTEGER_FIELD_1_VALUE = 1234;
+    protected static final int INTEGER_FIELD_2_VALUE = 2345;
+    protected static final int INTEGER_FIELD_3_VALUE = 3456;
+    protected static final int INTEGER_FIELD_4_VALUE = 4567;
+    protected static final String KEYWORD_FIELD_1 = "doc_keyword";
+    protected static final String KEYWORD_FIELD_1_VALUE = "workable";
+    protected static final String KEYWORD_FIELD_2_VALUE = "angry";
+    protected static final String KEYWORD_FIELD_3_VALUE = "likeable";
+    protected static final String KEYWORD_FIELD_4_VALUE = "entire";
+    protected static final String DATE_FIELD_1 = "doc_date";
+    protected static final String DATE_FIELD_1_VALUE = "01/03/1995";
+    private static final String DATE_FIELD_2_VALUE = "05/02/2015";
+    protected static final String DATE_FIELD_3_VALUE = "07/23/2007";
+    protected static final String DATE_FIELD_4_VALUE = "08/21/2012";
+    protected static final String INTEGER_FIELD_PRICE = "doc_price";
+    protected static final int INTEGER_FIELD_PRICE_1_VALUE = 130;
+    protected static final int INTEGER_FIELD_PRICE_2_VALUE = 100;
+    protected static final int INTEGER_FIELD_PRICE_3_VALUE = 200;
+    protected static final int INTEGER_FIELD_PRICE_4_VALUE = 25;
+    protected static final int INTEGER_FIELD_PRICE_5_VALUE = 30;
+    protected static final int INTEGER_FIELD_PRICE_6_VALUE = 350;
+    protected static final String BUCKET_AGG_DOC_COUNT_FIELD = "doc_count";
+    protected static final String BUCKETS_AGGREGATION_NAME_1 = "date_buckets_1";
+    protected static final String BUCKETS_AGGREGATION_NAME_2 = "date_buckets_2";
+    protected static final String BUCKETS_AGGREGATION_NAME_3 = "date_buckets_3";
+    protected static final String BUCKETS_AGGREGATION_NAME_4 = "date_buckets_4";
+    protected static final String KEY = "key";
+    protected static final String BUCKET_AGG_KEY_AS_STRING = "key_as_string";
+    protected static final String SUM_AGGREGATION_NAME = "sum_aggs";
+    protected static final String SUM_AGGREGATION_NAME_2 = "sum_aggs_2";
+    protected static final String AVG_AGGREGATION_NAME = "avg_field";
+    protected static final String GENERIC_AGGREGATION_NAME = "my_aggregation";
+    protected static final String DATE_AGGREGATION_NAME = "date_aggregation";
+    protected static final String CLUSTER_SETTING_CONCURRENT_SEGMENT_SEARCH = "search.concurrent_segment_search.enabled";
+    private static final String VALUE = "value";
+    private static final String RELATION = "relation";
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
It is a search call therefore can you make this update cluster settings call to happen just once at the start of test execution. Currently this call will made the number of times the tests are run.

Reply:
will try to use @BeforeClass instead of @Before